src/workspace-data/data-table/shared/RenameColumnModal.ts

```diff
@@ -44,18 +44,14 @@ export const RenameColumnModal = (props: RenameColumnModalProps): ReactNode => {
   const gcpColumnNameErrors =
     dataProvider.providerName === 'Entity Service'
       ? validate.single(newAttributeName, {
+          exclusion: {
+            within: ['name', 'entityType', `${entityType}_id`],
+            message: `Column name cannot be "name", "entityType" or "${entityType}_id".`,
```

Comment:
nit: these two lines could probably be coupled together a bit more (to avoid drift), maybe extract a variable with the list of excluded fields, then use that for generating the message?